app/src/sizes.sass

```diff
@@ -3,3 +3,6 @@ $navbarheight: 40px
 
 $small-space: 4px
 $medium-space: 8px
+
+
+$big-space: 45px
```

Comment:
$small-space and $medium-space are mostly spaces between borders or lines. If the font sizes change these spaces should remain the same. But if the border widths change then these spaces may change. On the contrary, it seems to me that the space between the dialog message and the button bar must be smaller if the font size is smaller and bigger if the font is bigger. Therefore, the size of this space should be written in ex or em unit. Moreover, I'm not convinced that the exact same space should be reused throughout the application. Hence I don't think it's a good candidate for an application-wide variable.